.gitconfig

```diff
@@ -0,0 +1,2 @@
+[core]
+	ignorecase = false
```

Comment:
Hmm... wonder why this needs to be set to `false` explicitly, if this is the default; git-config(1) says:

> The default is false, except git-clone(1) or git-init(1) will probe and set core.ignoreCase true if appropriate when the repository is created.

Reply:
Could the "where appropriate" be the issue when it detects a case-insensitive platform?

Reply:
I could try reverting the commit to see if it was just coincidence that it passed after I added it? idk

Reply:
If this helps devs using Mac OS X I'm fine with the change as it is.

Reply:
No it didn't change my local configuration, and I'm even in doubt now if GitHub actually uses the file, or that it was sheer luck that pushing again solved the Windows linter; I rebased, and removed the commit to confirm that suspicion